network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -543,27 +654,7 @@ def get_context(self, request, *args, **kwargs):
             products = ProductPage.objects.all()
         else:
             products = ProductPage.objects.live()
-
-        # TODO:
-        # Sort the products by their creepiness level. Example code below taken
-        # from buyersguide/views.py
-        # def get_average_creepiness(product_dict):
-        #     try:
-        #         votes = product_dict['votes']
-        #         creepiness = votes['creepiness']
-        #         avg = creepiness['average']
-        #         return avg
-        #     except TypeError:
-        #         pass
-        #     except AttributeError:
-        #         pass
-
-        #     return 50
-        # products = cache.get_or_set(
-        #     'sorted_product_dicts',
-        #     lambda: sorted([p.to_dict() for p in Product.objects.all()], key=get_average_creepiness),
-        #     86400
-        # )
+        products = sorted(products, key=lambda p: p.average_vote_count)
```

Comment:
are we reintroducing the `cache.get_or_set` call? because 10 queries vs. 650 queries is _kind of a big deal_ =D

Reply:
Yikes yes 650 is a big deal. Im sure this will be less when we use Pages but you're still right. Ill apply caching.